client/src/app/components/page/entry/entry.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
ok, lets delete those **.spec** files.
we don't need them for now.

Reply:
Spec file deleted.